js/ui/components/networkAgent.js

```diff
@@ -151,6 +168,16 @@ const NetworkSecretDialog = new Lang.Class({
         this._updateOkButton();
     },
 
+    _addKbdLayoutSwitcher: function() {
+        this._inputSourceIndicator = new InputSourceIndicator(this, false);
+        this._manager = new PopupMenu.PopupMenuManager({ actor: this._inputSourceIndicator.container });
+        this._manager.addMenu(this._inputSourceIndicator.menu);
```

Comment:
The `PopupMenuManager` is never used outside of this function, so you should not need to save it as a private member in `this._manager`